src/py/flwr/server/superlink/fleet/vce/vce_api_test.py

```diff
@@ -166,7 +166,7 @@ def start_and_shutdown(
         backend_name=backend,
         backend_config_json_stream=backend_config,
         state_factory=state_factory,
-        working_dir=working_dir,
+        app_dir=working_dir,
```

Comment:
Should this be renamed as well?
```suggestion
        app_dir=app_dir,
```